bin/ce

```diff
@@ -112,6 +112,17 @@ def confirm_branch(release):
         if typed == branch:
             return True
 
+def wait_for_healthok(instance):
```

Comment:
The problem is that when you exit standby (which is when "sudo systemctl restart compiler-explorer" exits), CE is still starting up (not to mention SSH/communication fails for a long time after restart)
So if standby is exited there already, the loadbalancer healthcheck will say Nope and scale it out to be replaced by another instance (because exiting standby also re-increases capacity requirement)

For it to be safe to exit standby, is for the instance healthcheck to return Ok.
So if we wait for the CE startup to be completed, then it won't be scaled out when we take it out of standby.

Reply:
I see...I set `instance protection` on before I do anything which _should_ (so I thought) protect the instance from being scaled out. I wonder why this wasn't working? :thinking:

Reply:
it should according to documentation i think, especially with wait_for_elb_state(instance, 'healthy') to wait for healthy before that... maybe its an aws bug?

Reply:
or maybe it returns as healthy, only for then nginx to be stopped and restarted? I haven't checked for that

Reply:
Absolutely yes...I'm not against this at all :) We can do a full "stop. wait to go unhealthy. start. wait for healthy" all while it's in protected state and then hopefully it Can't Go Wrong™